chemprop/nn/predictors.py

```diff
@@ -297,3 +297,14 @@ def __init__(self, *args, spectral_activation: str | None = "softplus", **kwargs
                 )
 
         self.ffn.add_module("spectral_activation", spectral_activation)
+
+    def forward(self, Z: Tensor) -> Tensor:
+        Y = super().forward(Z)
+        Y = self.ffn.spectral_activation(Y)
+        return Y / Y.sum(1, keepdim=True)
+
+    def train_step(self, Z: Tensor) -> Tensor:
+        Y = super().forward(Z)
+        Y = self.ffn.spectral_activation(Y)
+        return Y / Y.sum(1, keepdim=True)
```

Comment:
It's best to avoid copying code between methods:
```suggestion
   train_step = forward
```
This assigns the `train_step` method to the `forward` method. It looks like attribute assignment because, mechanically, it's the same thing in python.

Reply:
I see. Thanks for explaining. I have committed the suggestion.